packages/investor-foxy/src/api/api.ts

```diff
@@ -132,8 +129,8 @@ export class FoxyApi {
       if (dryRun) return signedTx
       try {
         if (this.providerUrl.includes('localhost') || this.providerUrl.includes('127.0.0.1')) {
-          const sendSignedTx = await this.web3.eth.sendSignedTransaction(signedTx)
-          return sendSignedTx?.blockHash
+          const sendSignedTx = await this.provider.sendTransaction(signedTx)
+          return sendSignedTx?.blockHash ?? ''
```

Comment:
This is confirming to the `sendTransaction` return type - runtime logic didn't change here